src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubsWebJobsStartup.cs

```diff
@@ -1,13 +1,13 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using Microsoft.Azure.WebJobs.Extensions.EventHubs;
+using Microsoft.Azure.WebJobs.EventHubs;
 using Microsoft.Azure.WebJobs.Hosting;
 using Microsoft.Extensions.Hosting;
 
 [assembly: WebJobsStartup(typeof(EventHubsWebJobsStartup))]
 
-namespace Microsoft.Azure.WebJobs.Extensions.EventHubs
+namespace Microsoft.Azure.WebJobs.EventHubs
```

Comment:
Just mentioned this above -- but it looks like our extensions in the Extensions repo all have the `Extensions` name in there.